src/interfaces/IEjectionManager.sol

```diff
@@ -32,7 +32,7 @@ interface IEjectionManager {
      * @notice Ejects operators from the AVSs registryCoordinator under a ratelimit
      * @param _operatorIds The ids of the operators to eject for each quorum
      */
-    function ejectOperators(bytes32[][] memory _operatorIds) external;
+    function ejectOperators(bytes32[][] memory _operatorIds) external returns (uint256[] memory);
```

Comment:
Sorry for late feedback: can we return uint32 instead? There won't be so many operators to eject